### PR TITLE
Lpc module building

### PR DESCRIPTION
**Pull request by krystian-hebel**

### CODE REVIEW

**Review by 3mdeb-karolzmyslowski — APPROVED** [review]
This instruction is quite clear and understandable. 
One point, I could add to it is the result of this commands (how to check, test whether everything is correctly build, flashed). (currently, there is missing part of the final result, how the device should behave - I believe, doing these steps will make work done, but I could not prove it).

File: fpga/quickfeather.pcf [hunk 1]
```diff
@@ -0,0 +1,83 @@
+// [WLCSP PACKAGE]  WR42    set_io <signal name> <QFN_pin_num>
```
Comment (3mdeb-karolzmyslowski) [inline fpga/quickfeather.pcf:1]: @krystian-hebel Lines 1-39 look like comments. 
Some cleanup is needed, before final version.

**Review by 3mdeb-karolzmyslowski — APPROVED** [review]

File: fpga/quickfeather.pcf (hunk 1 above)
Comment (krystian-hebel) [reply fpga/quickfeather.pcf:1]: These are comments with instructions how to set up IO signals. These will be modified as the project progresses, so I'd rather keep it for easy reference, at least until real top level module is ready.

Comment (3mdeb-karolzmyslowski) [reply fpga/quickfeather.pcf:1]: I understand. 
It could be resolved.

**Review by 3mdeb-karolzmyslowski — APPROVED** [review]
Approved.

**Pull request merged**